product_docs/docs/biganimal/release/pricing_and_billing/index.mdx

```diff
@@ -21,15 +21,17 @@ Extreme high availability powered by EDB Postgres Distributed is now available i
 
 ## Cloud infrastructure costs
 
-Whether you're connecting directly to BigAnimal through your Microsoft Azure or AWS account, EDB doesn't bill you for cloud infrastructure, such as compute, storage, data transfer, monitoring, and logging.
+If you're connecting directly to BigAnimal through BigAnimal's cloud, EDB bills you for cloud infrastructure, such as compute, storage, data transfer, monitoring, and logging. Monthly estimates for BigAnimal's cloud infrastructure costs can be viewed while creating a cluster. 
```

Comment:
some essay above in this PR uses the term of `BigAnimal's cloud account`, but this essay uses `BigAnimal's cloud` everywhere, I think it would be nice to make it consistent.